Readme.md

```diff
@@ -117,7 +117,7 @@ client, _ := analytics.NewWithConfig(WRITE_KEY,
 			Interval:     30 * time.Second,
 			BatchSize:    100,
 			Verbose:      true,
-			Gzip:         0  // Enables Gzip compression - set to 1 to disable Gzip.
+			DisableGzip:  false  // Enables Gzip compression - set true to disable Gzip.
```

Comment:
We don't need the comment here. 
1. For the sake of consistency, no other arguments had comments
2. New variable name is self explanatory

Reply:
This is an example file. Not a library file.